lib/db/schema/patch-080-081.sql

```diff
@@ -0,0 +1,59 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE TABLE IF NOT EXISTS recoveryKeys (
+  uid BINARY(16) NOT NULL,
+  recoveryKeyId BINARY(64) NOT NULL,
+  recoveryData TEXT,
+  UNIQUE KEY (`uid`, `recoveryKeyId`)
```

Comment:
Since you intend to only allow one key per user, it might be simpler to just mark the `uid` column as the primary key, then you don't need this `UNIQUE KEY` constraint.

Alternately, if you want to allow multiple keys in the future, you could make `(uid, recoveryKeyId)` the primary key, and add a `UNIQUE KEY (uid,)` constraint to ensure there's only one per user.  Then if and when we want to allow multiple keys per user, we can just drop that constraint.